zebra-rpc/src/methods.rs

```diff
@@ -869,6 +994,23 @@ pub struct GetBlock(#[serde(with = "hex")] SerializedBlock);
 #[derive(Copy, Clone, Debug, Eq, PartialEq, serde::Deserialize, serde::Serialize)]
 pub struct GetBestBlockHash(#[serde(with = "hex")] block::Hash);
 
+/// Response to a `z_gettreestate` RPC request.
+///
+/// Contains the hex-encoded Sapling & Orchard note commitment trees, and their
+/// corresponding [`block::Hash`], [`Height`] and block time.
+#[derive(serde::Serialize)]
+pub struct GetTreestate {
+    #[serde(with = "hex")]
+    hash: block::Hash,
+    #[serde(with = "hex")]
+    height: SerializedHeight,
```

Comment:
Blocker: making it easier to serialize heights is a good idea. (1/2)

But the serialization format is an integer, not hex-encoded bytes:
https://zcash.github.io/rpc/z_gettreestate.html

I think this should just work?
```suggestion
    height: Height,
```

Reply:
That's correct. I forgot that the height is represented just as an integer in the RPC response.